src/Dictionaries/RangeDictionarySource.h

```diff
@@ -14,170 +14,241 @@
 namespace DB
 {
 
-template <typename RangeType>
+enum class RangeDictionaryType
+{
+    simple,
```

Comment:
Should start with a capital according to our code style. Same for another enum above.

Reply:
@akuzm agreed, will fix.